test/styled/table.test.ts

```diff
@@ -283,15 +283,17 @@ describe('styled/table', () => {
   })
 
   describe('edge cases', () => {
-    fancy
-    .stdout()
-    .end('ignores header case', output => {
-      cli.table(apps, columns, {columns: 'iD,Name', filter: 'nAMe=supertable-test', sort: '-ID'})
-      expect(output.stdout).to.equal(` ID  Name${ws.padEnd(14)}
- ─── ─────────────────${ws}
- 321 supertable-test-2${ws}
- 123 supertable-test-1${ws}\n`)
-    })
+// Why is this a use case we need to check?
```

Comment:
So that someone doesn't accidentally make `--filter` case sensitive.

Reply:
So now filter is case sensitive, how much of a problem that is?